doc/QuickStart.md

```diff
@@ -167,7 +167,13 @@ equipment-player-* 	preLoad 	int 	1 	If 1, data pages preloaded with file conten
 equipment-player-* 	fillPage 	int 	1 	If 1, content of data file is copied multiple time in each data page until page is full (or almost full: on the last iteration, there is no partial copy if remaining space is smaller than full file size). If 0, data file is copied exactly once in each data page.
 ```
 
-Another option is to use the program `o2-raw-file-reader-workflow` in O2 and described [here](https://github.com/AliceO2Group/AliceO2/tree/dev/Detectors/Raw#raw-data-file-reader-workflow). One can for example produce raw data with some Simulation workflow, or record it with `readout.exe`.
+
+#### Getting real data from the O2 RawDataFileReader
+
+Another option to read a raw data file, produced by Simulation or recorded with `readout.exe` per instance, is to use the program `o2-raw-file-reader-workflow` in O2 as described [here](https://github.com/AliceO2Group/AliceO2/tree/dev/Detectors/Raw#raw-data-file-reader-workflow) (the config file is described [earlier in the page](https://github.com/AliceO2Group/AliceO2/tree/dev/Detectors/Raw#rawfilereader)). 
+```
+	o2-raw-file-reader-workflow --conf myConf.cfg | o2-qc --config json://${QUALITYCONTROL_ROOT}/etc/readout.json
```

Comment:
please delete unnecessary indentation, it requires the reader to scroll horizontally to see the full command

Reply:
correct, my editor hid that from me.